src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -92,7 +60,6 @@ void CircuitBreakerTriggered(object state)
         static TimeSpan NoPeriodicTriggering = TimeSpan.FromMilliseconds(-1);
         static ILog Logger = LogManager.GetLogger<RepeatedFailuresOverTimeCircuitBreaker>();
 
-        TimeSpan delayAfterFailure;
```

Comment:
Yup, not used. Downstreams might have used it but they should be copy pasting this class in that case. The core shouldn't be a utils lib